src/release.js

```diff
@@ -95,9 +99,23 @@ module.exports = async function ({ github, context, inputs }) {
   try {
     const opticToken = inputs['optic-token']
     const npmToken = inputs['npm-token']
+    const provenance = /true/i.test(inputs['provenance'])
```

Comment:
minor: for boolean inputs @actions/core contains a utility (but I believe what you're doing here works just fine and I'm fine with it

Reply:
Maybe that's better done as a small separate PR since it affects code out of scope of this feature.